CONTRIBUTING.md

```diff
@@ -1,4 +1,9 @@
-# 💥 How to Contribute
+# 💥 Prerequisite Skills to Contribute.
+- [node.js](https://nodejs.org/en/)
+- [Git](https://git-scm.com/)
+
```

Comment:
I think we should also mention react https://reactjs.org/

Reply:
mentioned react